tests/unit/amo/utils/test_addons.js

```diff
@@ -183,50 +183,157 @@ describe(__filename, () => {
     });
   });
 
-  describe('getPromotedCategory', () => {
-    it('returns null if the addon is not promoted', () => {
+  describe('getPromotedCategories', () => {
+    it('returns the empty list if the addon is not promoted', () => {
       const addon = createInternalAddonWithLang({
         ...fakeAddon,
-        promoted: null,
+        promoted: [],
```

Comment:
You need to keep this test with `null` for backward compatibility.

Reply:
I had moved it to a different test on line 316